egs/gale_arabic/s5b/local/chain/run_tdnn.sh

```diff
@@ -1 +1 @@
-tuning/run_tdnn_1a.sh
\ No newline at end of file
+tuning/run_tdnn_1b.sh
```

Comment:
The old (out-dated) tdnn_1a recipe should be either deleted, or you can clearly state the difference between 1a and 1b recipes, and add comparison of the results in the 1d recipe. Also it'll be great if you can add a new version of the old tdnn_lstm recipe (producing 17.64 WER which is competitive), to see whether you get some improvements (I suspect not much though). Finally you may want to update s5b/RESULTS to reflect the best tdnn/tdnn-lstm numbers.

Reply:
ok, thank you I will do it.